pkg/dao/snapshots_mock.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by mockery v2.43.0. DO NOT EDIT.
+// Code generated by mockery v2.42.3. DO NOT EDIT.
```

Comment:
i think since this would downgrade the version, we typically wouldn't commit them.  Can you regenerate them with a newer mockery?

Reply:
oh woops! fixed. using the make command to generate the mocks doesn't update the version if it's already installed in ./release, this should be fixed too